source/stylesheets/pages/_home.scss

```diff
@@ -113,34 +120,39 @@ section#about {
       flex-direction: row;
     }
   }
-  ion-icon {
-    display: block;
-    margin: 0 auto;
-    font-size: 44px;
+  .close {
+    appearance: none;
+    border: none;
+    background: transparent;
```

Comment:
This is the same button cleanup that I used earlier in the PR. I converted the close icon to a button as well, and added your `showHide()` method to the click handler.